fem/fespace.cpp

```diff
@@ -791,8 +791,14 @@ const Operator *FiniteElementSpace::GetElementRestriction(
 {
    // Check if we have a discontinuous space using the FE collection:
    const L2_FECollection *dg_space = dynamic_cast<const L2_FECollection*>(fec);
-   if (dg_space) { return NULL; }
-   // TODO: support other DG collections.
```

Comment:
I don't think this TODO was addressed here, so this comment should not be removed.